### ISSUE TITLE
How to acquire original training and test dataset

### ISSUE THREAD

**Issue by youhanmo** [lgsvl/lanefollowing#8]
Hi,Team

I was wondering how to acquire your original training set and test set used for training the model 
lanefollowing/ros2_wc/src/lane_following/model/model.h5

I did see it in the repository.

Thanks

**Comment by martins-mozeiko** [comment]
Following documentation page explains how this model is obtained: [How to Collect Data and Train Your Own Model with LGSVL Simulator](https://www.lgsvlsimulator.com/docs/lane-following/#how-to-collect-data-and-train-your-own-model-with-lgsvl-simulator)
